### PR TITLE
opentabletdriver: Rebuild against 8.0.6

### PR DESCRIPTION
**Pull request by nelsontkq**
**Summary**

Previously built binaries are corrupted. Attempting to build against newest .net to fix.

**Test Plan**

 - use tablet
 - ensure GUI works
 - keymap using tablet button

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

**Review by EbonJaeger — COMMENTED** [review]
Was this rebuilt after changing the `component`? It's different between the `package.yml` and the `pspec` file.

**Comment by nelsontkq** [comment]
@EbonJaeger rebuilt!

File: packages/o/opentabletdriver/pspec_x86_64.xml [hunk 1]
```diff
@@ -7,18 +7,18 @@
             <Email>nelson@truran.dev</Email>
         </Packager>
         <License>LGPL-3.0-only</License>
-        <PartOf>PLEASE FILL ME IN</PartOf>
+        <PartOf>multimedia.graphics</PartOf>
         <Summary xml:lang="en">An open source tablet driver</Summary>
-        <Description xml:lang="en">PLEASE FILL ME IN
+        <Description xml:lang="en">OpenTabletDriver is an open-source, user-friendly tablet driver designed to provide support for various drawing tablets on Linux. It offers advanced configuration options, customizable settings, and robust performance, ensuring a seamless experience for artists and designers.
```
Comment (nelsontkq) [inline packages/o/opentabletdriver/pspec_x86_64.xml:12]: I'm not sure why this one wasn't filled in before :thinking:

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks!

**Pull request merged**